build.gradle

```diff
@@ -56,6 +60,15 @@ if (project.hasProperty("maxParallelForks")) {
   ext.maxParallelForks = 3
 }
 
+animalsniffer {
+  ignoreFailures true // https://github.com/gradle/gradle/issues/1305
```

Comment:
It's failing for me with the version specified in the wrapper (3.4.1) and actually even with 3.5.